src/css_serializer.erl

```diff
@@ -0,0 +1,99 @@
+-module(css_serializer).
+
+-export([serialize_rules/1, serialize_declarations/1]).
```

Comment:
Because declarations, rules, and tokens all look different from each other, we need only one `serialize/1` function exported and match the different things in the function head like:
```erlang
serialize(CSS) when is_list(CSS) ->
  << <<(serialize(E))/binary, " ">> || E <- CSS >>;
serialize({rule, Prelude, Block}) ->
  <<(serialize(Prelude))/binary, ${, (serialize(Block))/binary, $}>>;
serialize({{decl, important}, Attr, Val}) ->
  <<(serialize(Attr))/binary, $:, (serialize(Val))/binary, " !important", $;>>;
serialize({{decl, _Important}, Attr, Val}) ->
  <<(serialize(Attr))/binary, $:, (serialize(Val))/binary, $;>>;
% etc
```